docs/source/ops.rst

```diff
@@ -11,7 +11,15 @@ torchvision.ops
 
 .. autofunction:: nms
 .. autofunction:: roi_align
+.. autofunction:: ps_roi_align
 .. autofunction:: roi_pool
+.. autofunction:: ps_roi_pool
+.. autofunction:: deform_conv2d
 
 .. autoclass:: RoIAlign
+.. autoclass:: PSRoIAlign
 .. autoclass:: RoIPool
+.. autoclass:: PSRoIPool
+.. autoclass:: DeformConv2d
+.. autoclass:: MultiScaleRoIAlign
+.. autoclass:: FeaturePyramidNetwork
```

Comment:
I was thinking that those two were still a bit experimental, but enough time has passed already so I think we can add those as well